pyccl/core.py

```diff
@@ -39,6 +39,18 @@
     'watson':   lib.watson
 }
 
+emulator_neutrinos_types = {
+	'strict': 	lib.strict,
+	'equalize': lib.equalize
+}
+
+mnu_types = {
+	'list': lib.mnu_list,
+	'sum': lib.mnu_sum,
+	'sum_inverted': lib.mnu_sum_inverted,
```

Comment:
Can probably just call these 'inverted' and 'equal'.

Reply:
I did it this way because I feel like it might be confusion to say just 'inverted' and 'equal' - the user might think they are specifying that they need to pass a list of masses in agreement with the inverted hierarchy. This way they know they are passing a sum. Do you feel strongly about changing this?